examples/channels-for-correlation/channels_correlation.bal

```diff
@@ -0,0 +1,40 @@
+import ballerina/http;
```

Comment:
Could you also please verify if the file names are correct? AFAIK the file names should match the folder name.

Reply:
They are always different, folders has '-' and bal files has '_'. You mean the words?

Reply:
I changed the words to be similar.